entity_client.go

```diff
@@ -474,6 +528,13 @@ func (c MockEntityClient) Query(query string, filter QueryFilter) ([]Entity, err
 	return nil, nil
 }
 
+func (c MockEntityClient) Get(id string) (Entity, error) {
```

Comment:
Possible to move MockEntityClient to the test files?

Reply:
That's a good idea and I tried it based on your suggestion, but it requires quite a bit of refactoring. I'm going to leave it as a possible future consideration.

The problem is that MockEntityClient depends on types defined in the etre package (etre.Entity, etre.Set, etc.). But entity_client and entity_client_test are also in the etre package. So if mock depends on etre then the tests in etre can't use the mock because of the circular dependency.